bigbluebutton-client/resources/prod/lib/kurento-extension.js

```diff
@@ -4,6 +4,13 @@ var isChrome = !!window.chrome && !isOpera;
 var isSafari = navigator.userAgent.indexOf("Safari") >= 0 && !isChrome;
 var kurentoHandler = null;
 
+// TODO sugestions on where to store this
```

Comment:
I think it's good for the moment. We'll devise a way to properly handle errors when we make the SFU API.
Also, I'll integrate this with the review I'm doing on error handling at `bbb-webrtc-sfu`.